<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainGrid">
    <item row="1" column="1">
     <widget class="QCheckBox" name="recursiveCheck">
      <property name="toolTip">
       <string>Recursively get every page that is linked to any page linked to any page linked.....</string>
      </property>
      <property name="text">
       <string>Recursive</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="tryWikipediaSearchButton">
      <property name="text">
       <string>Get Info</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="mainLabel">
      <property name="text">
       <string>Enter Wikipedia search term in the box below</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="searchTerm">
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Search Box&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QTabWidget" name="tabsSummaryLinkList">
      <property name="toolTip">
       <string>Links to all the wikipedia pages that will be downloaded along with the initial one</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Summary</string>
       </attribute>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>431</width>
          <height>217</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>429</width>
           <height>215</height>
          </rect>
         </property>
         <widget class="QTextBrowser" name="summaryTextBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>431</width>
            <height>191</height>
           </rect>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Segoe UI'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>List of links to get</string>
       </attribute>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>431</width>
          <height>217</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>429</width>
           <height>215</height>
          </rect>
         </property>
         <widget class="QListWidget" name="listOfLinksBox">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>431</width>
            <height>191</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="saveFilesButton">
      <property name="toolTip">
       <string>Save Database</string>
      </property>
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="filePathLabel">
      <property name="text">
       <string>This is the path that they will be saved to</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="saveDirectoryButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
